### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by shenef**

### CODE REVIEW

File: src/zoktai/any.md [hunk 1]
```diff
@@ -2,55 +2,67 @@
...
 
+This guide assumes you are playing on the japanese 1.0 release of Boktai 2.  
+It will likely work fine for other releases but some strategies like RNG manipulation will need different timings.  
 Directions for paths and exits will use ↖️, ↗️, ↙️ and ↘️.
 
+### Terminology
+
+- RNG manip: Short for "Random Number Generator Manipulation". Used to influence the randomness within the game to make things happen that are normally reliant on chance.
+- Soft reset: Pressing A + B + Start & Select at the same time reboots the game. This is useful for RNG manips and is faster when loading a save after a death.
+- STRG and AGIL: Strength and Agility. The second and third status point options. They influence your physical damage, movement speed and defense.
```
Comment (Drakodan) [inline src/zoktai/any.md]: STRG and AGIL are the third and fourth status point options

File: src/zoktai/any.md [hunk 2]
```diff
@@ -2,55 +2,67 @@
...
 
-Set Skip BIOS to FALSE and RTC to FALSE in the GBA Settings (Requires v2.6.3+):
-<img class="center" src="./assets/images/emu_settings.webp">
+- Requires BizHawk 2.6.3 or newer.
+- Set "Skip BIOS" and "RTC" to False in the GBA Settings.
 
-Start a new game and create a save file after killing the tutorial Zombie with the Sol de Vice.  
-Create a save file (This is required to access the Link menu) and soft-reset (A+B+Start+Select).
+Setting "Skip BIOS" is mandatory as this creates parity with actual hardware.  
+Disabling RTC is optional but recommended since it simulates an empty cartridge battery.
```
Comment (Drakodan) [inline src/zoktai/any.md]: Recommend not describing RTC as optional, having control of the ingame clock is integral to forcing the coffins to teleport.

File: src/zoktai/any.md (hunk 1 above)
Comment (shenef) [reply src/zoktai/any.md]: good catch, I even pulled out the game to make sure and still made that error

File: src/zoktai/any.md [hunk 3]
```diff
@@ -2,55 +2,67 @@
...
-<img class="center" src="./assets/images/system_time.webp">
+![](./assets/images/system_time.webp)
 
 Use the default region selection.
 
-## Setting up the Dark Boy password
+Select new Game and enter the password for the "Dark Boy" title.  
+This title grants +2 STRG, +2 AGIL, a ![][icon_power_nut] Power Nut and a ![][icon_bearnut] Bearnut.
+
+Use the characters to the right of the numbers:
```
Comment (Drakodan) [inline src/zoktai/any.md]: If this is the old image, the characters to the left of the numbers are used

File: src/zoktai/any.md [hunk 4]
```diff
@@ -87,13 +99,15 @@ Backtrack and take the stairs, kill all Ghouls as you proceed to reach **Lvl 4**
 In the 5th room, knock on the wall near the center of the room, then hug the right wall to avoid the Zombie.  
 Use ![][icon_flame] Flame to light the torches.
 
-### <span class="puzzle">Lamp Puzzle</span>
+### <span class="puzzle">Torch Puzzles</span>
```
Comment (Drakodan) [inline src/zoktai/any.md]: Puzzle header described as "torch", main body text still reads "lamp", recommend standardizing terms.

File: src/zoktai/any.md [hunk 5]
```diff
@@ -87,13 +99,15 @@ Backtrack and take the stairs, kill all Ghouls as you proceed to reach **Lvl 4**
...
-Use ![][icon_flame] Flame to light the torches, and hammer on levers to proceed.  
-After the dialogs, put all stat points into STRG.
+In the next rooms, use ![][icon_flame] Flame with ![][icon_gladius] Sword to light torches and ![][icon_club] Club on levers to proceed.  
+After the Otenko dialog, put all stat points into STRG.
 
-### <span class="trap">Trap:</span> Skeletons
+### <span class="trap">Skeletons Trap</span>
 
 - Use ![][icon_sol] Sol and hit the Skeletons from behind.
 - Stand on the third Skeleton's position as it dies in case you get a random drop.
```
Comment (Drakodan) [inline src/zoktai/any.md]: Remove this line for Short Sword RNG Manip route. With no random weapon drops, the Short Sword goes into inventory slot 3, which is the optimal spot for Katana Manip later.

File: src/zoktai/any.md (hunk 4 above)
Comment (shenef) [reply src/zoktai/any.md]: I'm using "Lamp" for the triggers you can walk against to activate them and "torch" for the ones you have to ignite with Flame.

**Pull request merged**